tests/control/PlanarArmControllerTest.cpp

```diff
@@ -1,13 +1,61 @@
 #include "../../src/control/PlanarArmController.h"
 
+#include "../../src/Constants.h"
+
+#include <iostream>
+
 #include <catch2/catch.hpp>
 
 using namespace Catch::literals;
 using namespace control;
 using namespace std::chrono_literals;
 
+constexpr double PI = M_PI;
+
+std::string toString(const Eigen::Vector2d& pose) {
+	std::stringstream ss;
+	ss << "(" << pose(0) << ", " << pose(1) << ")";
+	return ss.str();
+}
+
+void assertApprox(const Eigen::Vector2d& p1, const Eigen::Vector2d& p2, double dist = 1e-5,
+				  double angle = 1e-5) {
+	Eigen::Vector2d diff = p1 - p2;
+	bool distEqual = diff.norm() <= dist;
+
+	if (!distEqual) {
+		std::stringstream ss;
+		ss << "Expected: " << toString(p1) << ", Actual: " << toString(p2);
+		INFO(ss.str());
+	}
+
+	REQUIRE(distEqual);
```

Comment:
Catch2 prints out the evaluated expression in the `REQUIRE`, so we'll get better logs if we do `REQUIRE(diff.norm() <= dist)`